mephisto/client/review/review_server.py

```diff
@@ -85,10 +89,67 @@ def consume_data():
             ready_for_next.wait()
         finished = True
 
+    def consume_all_data(page, results_per_page):
+        limit = (
+            results_per_page
+            if (
+                results_per_page is not None
+                and isinstance(results_per_page, int)
+                and results_per_page > 0
+            )
+            else RESULTS_PER_PAGE_DEFAULT
+        )
```

Comment:
it might be cleaner to replace these Lines 92-101 here with this, using a default parameter value & assertion, e.g.:

```python
def consume_all_data(page, results_per_page = RESULTS_PER_PAGE_DEFAULT):
    assert type(results_per_page) is int, "results_per_page should be an int"
```